omnipaxos/src/lib.rs

```diff
@@ -24,6 +24,8 @@ pub use omni_paxos::*;
 
 /// The core replication algorithm of OmniPaxos.
 pub(crate) mod sequence_paxos;
+/// The interface between sequence paxos and the backend storage.
+pub(crate) mod internal_storage;
```

Comment:
At some point in the future, we will also try to fix some of the redundancies in `StateCache` and `LeaderState`, so I think it's better to further separate the different structs. I prefer we create a `storage` module with:
- `mod.rs` for `Storage`
- `internal_storage.rs` for `InternalStorage`
- `state_cache.rs` for `StateCache`